src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
+@mixin cell-propertis ($cell-width, $cell-height, $bg-color) {
+  width: $cell-width;
+  height: $cell-height;
+  background-color: $bg-color;
+}
+
+$space-size: 1px;
+$color-hover: pink;
+$start-day:
+  mon 1,
+  tue 2,
+  wed 3,
+  thu 4,
+  fri 5,
+  sat 6,
+  sun 7;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $space-size;
+  width: 720px;
+  margin: 0 auto;
+  padding: 10px;
```

Comment:
1. A better idea is not to hardcode the `width` but calculate it based on cell width, gaps, padding, etc
2. For some reason actual width of the calendar is `740px` - can you spot why?
<img width="1737" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/1983879/01acc6d9-78ff-403a-93cb-d111c1470ad7">